rotkehlchen/tests/db/test_db.py

```diff
@@ -33,7 +33,7 @@
     DEFAULT_UI_FLOATING_PRECISION,
     ROTKEHLCHEN_DB_VERSION,
     DBSettings,
-    ModifiableDBSettings,
+    ModifiableDBSettings, DEFAULT_ACCOUNTING_METHOD,
```

Comment:
next line. Running isort and the linting tets should do that for you
```suggestion
    ModifiableDBSettings,
    DEFAULT_ACCOUNTING_METHOD,
```

Reply:
done